editor/property_editor.cpp

```diff
@@ -4394,7 +4394,7 @@ PropertyEditor::PropertyEditor() {
 	use_filter = false;
 	subsection_selectable = false;
 	property_selectable = false;
-	show_type_icons = EDITOR_DEF("interface/editor/show_type_icons", false);
+	show_type_icons = false;
```

Comment:
At the very least there should be a comment explaining that it's not working and has to be ported to the new system. If we're just removing the option, we might just as well remove every reference to `show_type_icons`, as is it's dead code.

Reply:
So, should I remove it or leave a comment ?

Reply:
I guess leave a comment, this was a relatively popular feature (#15505) so I guess someone might want to fix their implementation in the new property editor.

Reply:
Ok, I leave a comment